pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -888,6 +881,44 @@ func (e *etcd) Scale(ctx context.Context, replicas int32) error {
 	return e.client.Patch(ctx, etcdObj, patch)
 }
 
+func (e *etcd) RolloutPeerCA(ctx context.Context) error {
+	if !e.nodeSpread {
+		return nil
+	}
+
+	etcdPeerCASecret, found := e.secretsManager.Get(v1beta1constants.SecretNameCAETCDPeer)
+	if !found {
+		return fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCAETCDPeer)
+	}
+
+	_, err := controllerutils.GetAndCreateOrMergePatch(ctx, e.client, e.etcd, func() error {
+		e.etcd.Annotations = map[string]string{
```

Comment:
Does it make sense to not add the operation annotation if `e.etcd.Spec.Etcd.PeerUrlTLS.TLSCASecretRef` is already set to the expected value (`etcdPeerCASecret`)?